### PR TITLE
Clarified expectations for Veertu recommendation.

### PR DESCRIPTION
**Pull request by jaydrogers**
Based off of the discussion in Issue #33, I figured this would be good to have in there to clarify a few things for the reader.

### CODE REVIEW

**Comment by KrauseFx** [comment]
Deploy preview for *ios-factor* ready!

Built with commit 3d3f6af9775dcdd01cac5d67f3b4556fea1588bc

https://deploy-preview-34--ios-factor.netlify.com

File: _i18n/en/factors/deployment.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ In an ideal world, to release an app update you would
 - automatically install all dependencies (e.g. Xcode and [CocoaPods](https://cocoapods.org))
 - run the deployment script (e.g. [fastlane](https://fastlane.tools))
 
-Unfortunately, due to the fact that Xcode has to run on macOS, and macOS virtualization comes with technical as well as legal challenges, we can't use this approach ([rdar://40669395](https://openradar.appspot.com/radar?id=4929082424819712)). One third party tool that makes it possible to have consistent macOS environments is [Veertu](https://veertu.com/).
+Unfortunately, due to the fact that Xcode has to run on macOS, we can't use this approach ([rdar://40669395](https://openradar.appspot.com/radar?id=4929082424819712)). Running macOS in a virtual environment comes with technical as well as legal challenges. There is a 3rd party commercial solution called [Veertu](https://veertu.com/) that makes it possible to have consistent macOS environments. It still requires Apple hardware to run, but it may be a good option to help you manage your build environments.
```
Comment (KrauseFx) [inline _i18n/en/factors/deployment.md]: Let's simplify to follow a similar language style to the rest of the page 👍 

```
Unfortunately, due to the fact that Xcode has to run on macOS, we can't use this approach ([rdar://40669395](https://openradar.appspot.com/radar?id=4929082424819712)). Running macOS in a virtual environment comes with technical as well as legal challenges. There is a 3rd party commercial (closed source) solution called [Veertu](https://veertu.com/) that allows you to generate virtaul macOS environments on Apple hardware.
```

Thoughts?

Comment (jaydrogers) [reply _i18n/en/factors/deployment.md]: Sounds great to me! Thanks @KrauseFx!

Comment (KrauseFx) [reply _i18n/en/factors/deployment.md]: Do you want to update the PR to use the new language, so that you're listed as the author of this change? :)

**Comment by jaydrogers** [comment]
This is fixed now @KrauseFx! Sorry for not editing it sooner. Let me know if you think it needs anything else. Preview is here: https://deploy-preview-34--ios-factor.netlify.com/deployment

Thanks for everything that you do! 👍

**Comment by KrauseFx** [comment]
Thanks for your contribution @jaydrogers 👍

**Pull request merged**